graph/options.go

```diff
@@ -56,16 +54,17 @@ type VendorOptions struct {
 // Options are all supported graph generation options.
 type Options struct {
 	AccessibleNamespaces map[string]time.Time
-	Appenders            []appender.Appender
```

Comment:
I still think you should keep this appender.Appender, even if it is only a marker interface now.. in the future, it will provide a place to add things when needed. It also makes it more strictly typed (now it is back to just string, which is too generic for my taste).

Reply:
The thing is that this option is no longer an array of Appenders but rather Appender names.  Just the appenders requested via query params.  The appender names are then passed to the actual telemetry vendor (i.e. istio) where they actually get converted into the actual Appender instances supplied by the vendor impl.  I could make this more clear by adding something like:

type AppenderName string

Is that what you are looking for?

Reply:
`type AppenderName string`

yes, that would be better